geth/jail/handlers.go

```diff
@@ -89,6 +89,20 @@ func registerHandlers(jail *Jail, cell *JailCell, chatID string) error {
 	return nil
 }
 
+// makeAsyncSendHandler returns jeth.sendAsync() handler.
+func makeAsyncSendHandler(jail *Jail, chatID string) func(call otto.FunctionCall) (response otto.Value) {
+	return func(call otto.FunctionCall) (response otto.Value) {
+		go func() {
+			res := jail.Send(call)
+
+			// Deliver response if callback is provided.
```

Comment:
Just for understanding of `newResultResponse()`: Isn't it _when_ the callback is provided, not _if_?